restcomm/restcomm.testsuite/src/test/java/org/restcomm/connect/testsuite/telephony/DialRecordingS3UploadSecureTest.java

```diff
@@ -269,6 +274,116 @@ public synchronized void testDialClientAlice_BobDisconnects() throws Interrupted
 		verify(1, putRequestedFor(urlMatching("/s3/.*")));
 	}
 
+	@Test
+	@Category(FeatureAltTests.class)
+	public synchronized void testDialClientAlice_BobDisconnects_AndRemoveRecording() throws InterruptedException, ParseException, IOException {
```

Comment:
what is this test adding on top of the previous test in terms of purpose?

Reply:
@jaimecasero We need to test the following:
- Delete recording from Record verb
- Delete recording from Dial record
- Delete recording for recording uploaded to S3

And thus we have those 3 tests